modules/installing-ocp-agent.adoc

```diff
@@ -146,6 +151,8 @@ agent.iso  auth
 ----
 $ openshift-install agent create image
 ----
++
+NOTE: Red Hat Enterprise Linux CoreOS (RHCOS) supports multipathing on the primary disk, allowing stronger resilience to hardware failure to achieve higher host availability. Multipathing is enabled by default in the `agent.iso` image. The `rd.multipath=default` argument creates a default `/etc/multipath.conf` file and enables the multipathd service.
```

Comment:
The last sentence is accurate could be a bit ambiguous (we supply that argument in the ISO, but it could be interpreted as meaning that the user has to do something). We could change it to something like: "Multipathing is enabled by default in the `agent.iso` image, with a default `/etc/multipath.conf` configuration."